resources/gui/bottommenu.rml

```diff
@@ -66,21 +66,22 @@ docmanage.manager.loadDoc(docmanage.manager.InventoryFile)
 docmanage.manager.loadDoc(docmanage.manager.CharacterFile)
 docmanage.manager.loadDoc(docmanage.manager.SpellsFile)
 docmanage.manager.loadDoc(docmanage.manager.QuestFile)
-        </script>
```

Comment:
why did you modify this?

Reply:
Actually, we should just delete this whole directory now

Reply:
Well I was removing all the rml/py changes which were having merge conflicts, so some slipped. But yes it probably should be removed and used only for reference.